lms/templates/lti_launches/new_lti_launch.html.jinja2

```diff
@@ -1 +1,17 @@
-<iframe width="100%" height="100%" src="{{hypothesis_url}}" />
\ No newline at end of file
+<script>
+  function receiveMessage(event) {
+    if (event.origin !== "http://localhost:5000") {
+      return;
+    }
+
+    if (event.data !== "gimme_grant_token") {
+      return;
+    }
```

Comment:
Only respond to requests from the client for grant tokens. In the future the client may send other kinds of requests for other things. As noted in <https://github.com/hypothesis/client/pull/764#discussion_r214714586> we should probably implement a JSON-RPC message protocol here.